### ISSUE TITLE
unhandled error with icon

### ISSUE THREAD

**Issue by nmmik** [getcouped/eslint-plugin-react-google-translate#3]
Hi there!

Today we encountered a page crash caused by an icon that was not wrapped in a <span>:

```

  const iconRight = iconRightEl ? <i className={iconStyle}>{iconRightEl}</i> : null;

  return (
    <button className={className} {...htmlAttributes} disabled={isDisabled} role="button">
{children}
      {iconRight}
    </button>
  );`
```

During local development, we get the following error in the console:

![Image](https://github.com/user-attachments/assets/487bd6e1-b7f8-4e14-8c43-33d7ad6770a2)

Unfortunately, your linter doesn’t catch this issue.

It would be great if a future version could address this!

**Comment by alistair-coup** [comment]
We have just published an update adding type awareness to the plugin which should catch any existing issues like this.
